lib/zendesk_api/middleware/response/sanitise_response.rb

```diff
@@ -0,0 +1,13 @@
+module ZendeskAPI
+  module Middleware
+    module Response
+      class SanitiseResponse < Faraday::Response::Middleware
+        def on_complete(env)
+          unless env[:body].strip.empty?
+            env[:body] = env[:body].encode('UTF-8', 'binary', invalid: :replace, undef: :replace, replace: '')
```

Comment:
I think what you want is: `"€foo\xA0".encode(Encoding::UTF_8, invalid: :replace, undef: :replace, replace: '')`

because it keeps more characters and it is the same encoding as (hopefully) everything else we load
```
"€foo\xA0".encode(Encoding::BINARY, invalid: :replace, undef: :replace, replace: '')
=> "foo"
"€foo\xA0".encode(Encoding::UTF_8, invalid: :replace, undef: :replace, replace: '')
=> "€foo"
```